packages/rum/src/userActionCollection.ts

```diff
@@ -83,7 +83,7 @@ function newUserAction(lifeCycle: LifeCycle, type: UserActionType, name: string)
         name,
         startTime,
         type,
-        duration: endTime - startTime,
+        duration: msToNs(endTime - startTime),
```

Comment:
This seems problematic to me, because the `msToNs` convertion will be done here too: https://github.com/DataDog/browser-sdk/blob/e03888639ab60f1b55cb91043958af281a9329ea/packages/rum/src/rum.ts#L317

Like other lifecycle events `duration` properties are in milliseconds, I don't think we should change this here.

Reply:
my bad then